piwebapi_samples/Data_Analysis/README.md

```diff
@@ -0,0 +1,66 @@
+
+The problem here is that these control systems are not identical. They all have different cooling rates so all of them don't take exactly 2 hours to reach the set point. Suppose one of them (Unit A) might reach the set point of 70 deg F by 6 AM. Unit A is now running for an extra hour in order to maintain the temperature of the office when there is nobody in the office. Our motivation here is to minimize this extra time so that we can save our costs and save some wasted energy as well.
+
+### Problem Statement
+Our objective here is to predict the total time taken by each of the unit in the building to reach the setpoint so that building management systems can turn on the units as close as possible to 7 AM.
+
+In our earlier example, if we had predicted earlier (to some degree of accuracy) that the Unit A takes an hour to reach the set point, we would have turned it on by 6 AM instead of 5 AM which would have resulted us in saving that extra hour of wasted energy.
+
+We will be utilizing the data that we have and do some basic data analysis and predictive machine learning using Jupyter Notebook in order to predict the cooling time for different units.
+
+### Data Overview
+In this sample, we are denoting these control systems as two different `VAVCO` units which are represented as AF Elements. Each of these elements have different attributes which mimic an actual cooling unit such as `% cooling` and `Set Point Offset`.
+
+There are also eventframes of the template `VAVCO startup` which represent the start up events of the cooling unit. They have some attributes which capture the metadata on start time as well as end time. 
+
+All this data is available as part of `Building Data.xml`. There is a helpful utility which will import this database as well as create and populate the PI Points. Find more on how to use it in the next section.
+
+## Getting Started
+- Clone the GitHub repository
+- Open the `Data_Analysis` folder with your IDE (eg. Visual Studio Code)
+- Install the required modules by running the following command in the terminal : `pip install -r requirements.txt`
+
+### Setting up the AF database and the PI Data Archive
+- In the `Data_Analysis` folder, populate the values of `test_config.json` with your own system configuration. 
+For example:
+```
+	"PIWEBAPI_URL": "https://mydomain.com/piwebapi/",
+	"AF_SERVER_NAME": "AssetServerName",
+	"PI_SERVER_NAME": "PIServerName",
+	"AF_DATABASE_NAME": "AFDatabaseNAme",
+	"USER_NAME": "MyUserName",
+	"USER_PASSWORD": "MyUserPassword",
+	"AUTH_TYPE": "basic"
+```
+- `Building Example.xml` assumes that the PI Server is on the same system. Edit the `Building Example.xml` to replace all occurences of `localhost`to your PI Server.
+- Run `PIUploadUtility.sln` which imports the AF database from `Building Example.xml`, creates PI tags outlined in `tagdefinition.csv` and uploads the values in `pidata.csv` to PI Data Archive.
+- The 
```

Comment:
Feel like there is more to this sentence... (or maybe less)

Reply:
I don't understand, would you like it to be divided into multiple sentences?